sdk/iOS/src/MSCoreDataStore.m

```diff
@@ -65,6 +65,9 @@ -(id) getRecordForTable:(NSString *)table itemId:(NSString *)itemId asDictionary
     
     if (item && asDictionary) {
         
+        //Someone, please, test this. Solves the warning, but I don't know it it will work as intended.
+//        NSDictionary *result = [item dictionaryWithValuesForKeys:[[[item entity] attributesByName] allKeys]];
+        
```

Comment:
This change looks good. The next line in the if-block ([NSDictionary dictionaryWithDictionary:result]) is not needed after this change as 'result' is not NSKnownKeysDictionary.
This change will also take care of issue https://github.com/Azure/azure-mobile-services/issues/779.

I am curious to know what warning you were seeing.